docs/UserGuide.md

```diff
@@ -266,6 +283,23 @@ Format: `delMeeting i/INDEX`
 Examples:
 * `delMeeting i/2`
 
+#### 3.2.5 Editing a meeting: `editMeeting`
+
+Edits details of the specified meeting.
+
+Format: `editMeeting i/INDEX [d/DATE] [st/START TIME] [et/END TIME] [dn/DESCRIPTION]`
+
+* Edits information about the meeting at the specific `INDEX`.
+* The index refers to the index number shown in the displayed meetings' list.
```

Comment:
i think you should keep it consistent with what you have done for `editClient` above. ie `index` => `INDEX` and link the meeting list